src/plugins/es_ui_shared/static/forms/hook_form_lib/hooks/use_form.test.tsx

```diff
@@ -316,6 +318,40 @@ describe('useForm() hook', () => {
           name: defaultValue.user.name,
         },
       });
+
+      expect(formHook?.__getFormDefaultValue()).toEqual({
+        ...defaultValue,
+        subTitle: 'hasBeenOverridden',
+      });
+    });
+
+    test('should be updated with the UseField "defaultValue" prop', () => {
```

Comment:
This test is not related to the new API change but I realised it was missing so I added it 😊